### PR TITLE
check to show My stake and remove the extra check to show dialog when…

### PR DESCRIPTION
**Pull request by KarishmaBothara**
… no account exist on New stake page

### CODE REVIEW

File: packages/app-staking/src/NewStake/index.tsx [hunk 1]
```diff
@@ -127,7 +127,7 @@ function NewStake ({ className, isVisible }: Props): React.ReactElement<Props> {
       setOpenHelpDialog(false);
     }
 
-    const openAccountCheckingModal = !hasAccounts || stakedAccounts.length == allAccounts.length || unstakedAccounts.length === 0;
+    const openAccountCheckingModal = !hasAccounts || stakedAccounts.length == allAccounts.length;
```
Comment (KarishmaBothara) [inline packages/app-staking/src/NewStake/index.tsx:130]: noticed the flickering and removed this bit for now.. will think about proper solution for this

File: packages/app-staking/src/index.tsx [hunk 2]
```diff
@@ -19,13 +19,17 @@ import { toFormattedBalance } from "@polkadot/react-components/util";
...
+import {findStakedAccounts} from "@polkadot/app-staking/MyStake/utils";
+import {Api} from "@cennznet/api";
 
 export default function ToolboxApp ({ basePath }: Props): React.ReactElement<Props> {
     const { t } = useTranslation();
     const { api } = useApi();
     const { allAccounts, hasAccounts } = useAccounts();
     const { pathname } = useLocation();
     const allStashes = useStashIds();
+    const [ showMyStash, setshowMyStash ] = useState<boolean>(true);
```
Comment (holygits) [inline packages/app-staking/src/index.tsx:31]: `showMyStake`

File: packages/app-staking/src/index.tsx [hunk 3]
```diff
@@ -19,13 +19,17 @@ import { toFormattedBalance } from "@polkadot/react-components/util";
...
+    findStakedAccounts(api as Api, allAccounts).then(stakedAccounts => stakedAccounts.size === 0 ? setshowMyStash(false): setshowMyStash(true) );
```
Comment (holygits) [inline packages/app-staking/src/index.tsx:32]:
```suggestion
    findStakedAccounts(api as Api, allAccounts).then(stakedAccounts => setShowMyStake(stakedAccounts.size > 0));
```

**Review by holygits — APPROVED** [review]
lgtm!

**Pull request merged**